### ISSUE TITLE
Nginx uses removed file.sed state

### ISSUE THREAD

**Issue by copelco** [caktus/margarita#43]
The [server_names_hash_bucket_size](https://github.com/caktus/margarita/blob/master/nginx/init.sls#L8-L13) replacement doesn't work with Salt 2015.5.0. AFAICT, [file.sed](http://docs.saltstack.com/en/latest/ref/modules/all/salt.modules.file.html#salt.modules.file.sed) was deprecated in 0.17 with [file.replace](http://docs.saltstack.com/en/latest/ref/states/all/salt.states.file.html#salt.states.file.replace) as a replacement state ([release notes](http://docs.saltstack.com/en/latest/topics/releases/0.17.0.html#fixes-from-rc-to-release)). This worked for me:

```yaml
/etc/nginx/nginx.conf:
  file.replace:
    - pattern: "(# )?server_names_hash_bucket_size .+;"
    - repl: "server_names_hash_bucket_size 64;"
    - require_in:
      - service: nginx
```

However, I'm not sure what version we're targeting as this may break deployments running older versions of Salt.

**Comment by vkurup** [comment]
I think #42 fixes the same thing. There are enough breaking changes with salt over time that older projects should just pin margarita, of needed.

**Comment by mlavin** [comment]
Yes this has been fixed by #42 and should be merged.
